docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: "3.7"
```

Comment:
@vickywane this docker-compose recipe should be part of a setup for dev install. Either a sub-directory of this repo or a separate repo.

Reply:
@ivelin Going through the `docker-compose.yml` file, I can see there is no need for it. 

The existing `ambianic-start.sh` script is enough to run the edge device.